### PR TITLE
Update security.md

### PR DESCRIPTION
**Pull request by Ghosgt**
update zh translate

### CODE REVIEW

**Comment by tengqm** [comment]
@Ghosgt Please sign the CLA before proposing more PRs.

**Review by Arhell — COMMENTED** [review]
more info about CLA, check please
https://github.com/kubernetes/community/blob/master/CLA.md

**Comment by jimangel** [comment]
Hey there! @Ghosgt, looks like you haven't signed the CLA yet. Could I please have you do that? https://github.com/kubernetes/community/blob/master/CLA.md

File: content/zh/docs/reference/issues-security/security.md [hunk 1]
```diff
@@ -47,8 +47,8 @@ You can also subscribe to an RSS feed of the above using [this link](https://gro
 <!--
 We’re extremely grateful for security researchers and users that report vulnerabilities to the Kubernetes Open Source Community. All reports are thoroughly investigated by a set of community volunteers.
 -->
-我们非常感谢向 Kubernetes 开源社区报告漏洞的安全研究人员和用户。
-所有的报告都由社区志愿者进行彻底调查。
+我们非常感谢向Kubernetes开源社区报告漏洞的安全研究人员和用户。
```
Comment (mengjiao-liu) [inline content/zh/docs/reference/issues-security/security.md:50]:
```suggestion
我们非常感谢向 Kubernetes 开源社区报告漏洞的安全研究人员和用户。
```
Spaces need to be added between Chinese and English characters.

**Comment by mengjiao-liu** [comment]
By the way, your CLA signature verification is still incomplete. @Ghosgt

**Comment by tengqm** [comment]
CLA not signed for over two months. Please feel free to reopen when it is ready.
/close

**Pull request closed without merging**